docker-compose.override.yml

```diff
@@ -0,0 +1,24 @@
+# These overrides are useful for local development purposes and will be loaded
+# by default when executing `docker-compose up`.
+# Put custom config in local.env.
+
+version: "3"
+
+services:
+  assets:
+    command: npm run watch
+    volumes:
+      - ./assets:/assets
+      - ./volumes/assets/node_modules:/assets/node_modules
```

Comment:
I'm still learning docker. I don't understand what you are mounting here and why?

Reply:
I did this so I can have a `node_modules` directory on my macOS host that is different than the one in the container. Otherwise they'll conflict with each other, meaning you need to reinstall them over and over again if developing against both.